infrastructure/bin/atat-web-api.ts

```diff
@@ -32,13 +30,21 @@ if (app.node.tryGetContext("TicketId")) {
 const environmentName = normalizeEnvironmentName(environmentParam);
 const environmentId = lowerCaseEnvironmentId(environmentParam);
 
+// Delete all resources in developer-specific sandbox environments by default, while
+// using the default for the specially-named long-lived "sandbox" resources, used
+// for persistent IAM configuration, etc.
+// TODO?: Make dev/staging/prod not "special" names -- introduce a different flag to
+// signify whether something is temporary
+if (!["sandbox", "dev", "staging", "prod"].includes(environmentId)) {
```

Comment:
What about adding a new flag to indicate that this is a disposable env? That way we don't open ourselves up to prod getting destroyed because someone changed it from `prod` to `production`.

Reply:
I guess we could also write that as one regex if we wanted to:

```ts
/^(sandbox|dev|stage?|prod)/.test(environmentId)
```

Reply:
Checking for _starts with_ rather than _exact match_ does seem less problematic until that TODO is done.